### ISSUE TITLE
Price Including Tax snippet doesn't work on manually created orders

### ISSUE THREAD

**Issue by FreshPhil** [woocommerce/woocommerce#26067]
**Is your feature request related to a problem? Please describe.**
When using the [Prices including tax - Experimental Behavior](https://github.com/woocommerce/woocommerce/wiki/How-Taxes-Work-in-WooCommerce#prices-including-tax---experimental-behavior) snippet:

`add_filter( 'woocommerce_adjust_non_base_location_prices', '__return_false' );`

It works on orders placed on the front end by customers, however it does not work on manually created orders on the backend.

**Describe the solution you'd like**
Have the snippet work the same for both front-end and manually created (back-end) orders.

**Additional context**
2831326-zd

**Comment by hambergk** [comment]
Any development on this? We have a customer who creates orders mainly from the backend, so it would be paramount to get this working as soon as possible.

**Comment by FITZKRAWALLDO** [comment]
Any news on this? We need it really

Same question was already closed without a fix:
https://github.com/woocommerce/woocommerce/issues/19083

**Comment by dkiss** [comment]
any update on that? i am also questioning the label "enhancement". we are adding heavy numbers of orders through backend and the tax calculation is wrong each time.

maybe we are able to sponsor some development here to fix the issue.

**Comment by earlz-roman** [comment]
Hey. We are experiencing the same issue and it's very important for us to have it solved. Is there any chance to prioritize it?

**Comment by daigo75** [comment]
**Disclaimer**: I don't want to come across as a sales man, because it's not my intention. 

I solved the issue of fixed product prices (not shipping costs, due to https://github.com/woocommerce/woocommerce/issues/9320) in my Tax Display by Country plugin, which uses a different approach to keep prices fixed. During the testing, I had the chance to investigate the reason why the `woocommerce_adjust_non_base_location_prices` snippet doesn't work in that context. 

The root cause is that the filter is used by the cart object, during the calculation of cart totals. When one creates a manual order, the cart object is not used, as the items are added to the order by following a different process. Such process always extracts the product price exclusive of tax, adds it to the order, then the tax is calculated when the "Calculate Totals" or "Recalculate" buttons are clicked. In this scenario, the `woocommerce_adjust_non_base_location_prices` is not triggered.

To fix this behaviour in the core, it would be necessary to introduce that filter to the "add item" process followed by the manual order creation. As of today, the filter is completely absent (i.e. it's not malfunctioning, it's missing altogether).

**Comment by robertvanhaecke** [comment]
Hey. A client of mine is also experiencing this issue and wants this to be solved.
Is there a temporary fix or workaround that can be used to overcome this issue?

**Comment by xtra-dev** [comment]
Hi everyone, I studied all posts I could find regarding this issue the past hour, and the problem that orders created on the backend can't get filtered by 'woocommerce_adjust_non_base_location_prices' was raised initially almost 4 years ago.

I understand that tax calculation is complex but is there a way to vote, contribute, donate to get this feature implemented?

I am currently working on a client project and have to find a solution. Product prices must remain the same, no matter what tax rate is applied to orders (depending on the billing country of the customer).

@daigo75 thanks for sharing your insights after digging through the Wordpress core code. I looked up your plugin that you mentioned. Is it [this one](https://aelia.co/shop/tax-display-by-country-for-woocommerce/)? And could your plugin possibly provide a solution to get manually created orders on the backend display fixed gross prices, no matter what tax rate is applied?

**Comment by daigo75** [comment]
@xtra-dev yes, that's the plugin I mentioned earlier. As I explained in my earlier post, when you add a product to a manual order, WooCommerce always adds an item with the price exclusive of tax, and adds the tax during the calculation of totals. 

The Tax Display by Country plugin intercepts this logic. The he price exclusive of tax is recalculated so that, once taxes are added, the final value is the original price, inclusive of tax.

I can't post links to other plugins here, nor discuss them, as this is a support forum for WooCommerce. If you would like to continue the conversation, I would like to ask you to use the contact form on my site, and I will be happy to send you further details. Thanks.

**Comment by jamesckemp** [comment]
This issue has been automatically closed as part of the backlog cleanup announced in [Refining how we manage the WooCommerce backlog](https://developer.woocommerce.com/2026/05/29/backlog-cleanup/). It met the criteria outlined there. If this still affects a current version of WooCommerce, please follow the steps in that post. A fresh report with current reproduction steps is the fastest path to triage. If this is a feature request, please add it to the [WooCommerce feature request board](https://woocommerce.com/feature-requests/woocommerce/) so it can be considered alongside other requests.
